native/cocos/renderer/gfx-gles2/GLES2Wrangler.cpp

```diff
@@ -98,7 +99,12 @@ bool gles2wClose() {
 void *gles2wLoad(const char *proc) {
     void *res = nullptr;
     if (eglGetProcAddress) res = reinterpret_cast<void *>(eglGetProcAddress(proc));
-    if (!res) res = dlsym(libegl, proc);
+    auto sdkVersion = cc::BasePlatform::getPlatform()->getSdkVersion();
+    if (sdkVersion <= 23) {
```

Comment:
In earlier versions of Android (including Android 6.0 and below), libGLESv2.so contains both the OpenGL ES 2.x rendering API and the EGL functionality. This means that when you link libGLESv2, you can access both OpenGL ES 2.x functions (like glClear, glDrawArrays, etc.) and the EGL functions (like eglCreateContext, eglMakeCurrent, etc.) without needing to link libEGL separately.